controllers/rollingupgrade_controller.go

```diff
@@ -1054,20 +1077,16 @@ func (r *RollingUpgradeReconciler) UpdateInstance(ctx *context.Context,
 		r.info(ruObj, "termination waiter completed but node is still joined, will proceed with upgrade", "nodeName", nodeName)
 	}
 
-	// set the EC2 tag indicating the state to completed
-	err = r.setStateTag(ruObj, targetInstanceID, "completed")
```

Comment:
Why are we removing this?

Reply:
The instance is already terminated. So this call always fails

Reply:
You can still tag a terminated instance, this is required for the resume feature